pkg/controller/egress_router/egress_router_controller.go

```diff
@@ -182,7 +182,7 @@ func (r *EgressRouterReconciler) ensureEgressRouter(manifestDir string, namespac
 	data.Data["AllowedDestinations"] = router.Spec.Redirect.RedirectRules
 	data.Data["mode"] = router.Spec.Mode
 	data.Data["network_interfaces"] = router.Spec.NetworkInterface
-	data.Data["EgressRouterPodImage"] = os.Getenv("EGRESS_ROUTER_CNI_IMAGE")
+	data.Data["EgressRouterPodImage"] = "registry.redhat.io/openshift3/ose-pod"
```

Comment:
ok so this PR changes the removes the cidr checks since every destination detail will have an IP and not CIDR. However I do not understand why this line is changing? The pod image is not the CNI image but the old openshift-3 version image ?

Reply:
Hmm yeah that doesn't make sense to me I will keep the envvar maybe that was just for testing